### PR TITLE
chore: add gha tests

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Refs: OPS-9006

Uses silk tests in `tests/run.sh` instead of unit tests, so no need for code coverage.

Also, the functions in ScriptHandler.php don't seem to be required any more and the tests were failing as it couldn't autoload them, so I removed their invocation (but not the script itself, which should probably happen too.)

### CODE REVIEW

**Comment by lazysoundsystem** [comment]
Tests failed on 
```
 dial tcp: lookup docstore-test-site on 127.0.0.11:53: server misbehaving  
--- FAIL: silk (0.01s)
```
the same happened when I ran it locally, fixed by adding an entry for http://docstore.test to /etc/hosts.
Will find a fix.

**Comment by lazysoundsystem** [comment]
Okay, the issue was trying to keep the travis tests at the same time so we ended up with a naming conflict - without Travis is easier.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+            *Pusher: @${{ github.actor }}, Action: `${{ github.event_name }}`, Workflow: `${{ github.workflow }}`*
+          edit-mode: replace
+
+      - name: Slack Success Notification
+        id: slack_success
+        if: ${{ !env.ACT && success() }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: '${{ secrets.SLACK_CHANNEL }}'
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:213]:
```suggestion
          channel-id: '${{ vars.SLACK_CHANNEL }}'
```

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+            }
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+      - name: Slack Failure Notification
+        id: slack_failure
+        if: ${{ !env.ACT && failure() }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: '${{ secrets.SLACK_CHANNEL }}'
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:240]:
```suggestion
          channel-id: '${{ vars.SLACK_CHANNEL }}'
```

**Review by cafuego — APPROVED** [review]

**Pull request merged**